### ISSUE TITLE
Electron app doesn't support Windows 10 Narrator for screen reading

### ISSUE THREAD

**Issue by digitalnetbizz2** [electron/electron#11867]
* Electron version: 1.6.10
* Operating system: Windows 10

### Expected behavior
Launch an electron app like https://github.com/electron/electron-api-demos
Win+U and enable narrator
Tabbing around the screen.
Narrator to speak out when focus on any accessibility element in the screen in an Electron App.

### Actual behavior

No correct read-out.

### How to reproduce

On Windows 10
Launch an electron app like https://github.com/electron/electron-api-demos
Win+U and enable narrator
Tabbing around the screen.

**Comment by codebytere** [comment]
Given that this isn't currently on our roadmap, i'm going to go ahead and label this a `wontfix`. However, should someone determine a workable solution we would be very open to accepting a PR!

**Comment by brenca** [comment]
This is an upstream issue that should be resolved in `4-0-x`. The gist of the issue is that the narrator uses `UIAutomation` APIs to read things from the screen instead of the old `MSAA`, which is not yet fully implemented in `chromium`. `4-0-x` (or `chromium 69`) brings the implementation up to an - in my opinion - acceptable level, where now even the menu items can be read by the narrator (that was an issue before).
